tests/e2e/koperator_test.go

```diff
@@ -141,6 +196,62 @@ func requireInstallingKoperatorHelmChartIfDoesNotExist(
 		}
 
 		By("Verifying koperator pods")
-		requireRunningPods(kubectlOptions, "app.kubernetes.io/name", "kafka-operator")
+		//requireRunningPods(kubectlOptions, "app.kubernetes.io/name", "kafka-operator")
+	})
+}
+
+// requireUninstallingKoperator uninstall koperator Helm chart and removes Koperator's CRDs
+// the success of those operations.
```

Comment:
I think this line is missing something?

Reply:
fixed: [b905456](https://github.com/banzaicloud/koperator/pull/987/commits/b90545678eb4424ed28ad0f7e998b95cc3c26eaf)